src/app/tests/suites/TestUserLabelClusterConstraints.yaml

```diff
@@ -54,4 +54,4 @@ tests:
                   },
```

Comment:
Pre-existing, but could we use 17-char label and value here (with documentation, if any is needed, in comments, not the values) to test that the right boundary condition is being tested?  Because the values here are long enough they would have failed out in various ways even on the code before we added checking here.

Reply:
This one should probably be deleted - in the meantime a Userlabel test was added with exactly 17 chars, so having 2 tests performing the same check seems not that useful.